db/db_impl/db_impl.cc

```diff
@@ -2883,7 +2901,6 @@ void DBImpl::MultiGetCommon(const ReadOptions& read_options,
   bool should_fail = false;
   for (size_t i = 0; i < num_keys; ++i) {
     ColumnFamilyHandle* cfh = column_families[i];
-    assert(cfh);
```

Comment:
Should we keep this, since it's just debug code?

Reply:
Since we removed the fallback, cfh can actually be null. (If the user passes in null explicitly for cfh)